parser/SqlLexer.lpp

```diff
@@ -39,6 +39,8 @@ class Comparison;
 class ParseAssignment;
 class ParseAttribute;
 class ParseAttributeDefinition;
+class ParseBlockProperties;
```

Comment:
When changing the Lexer or Parser sources, make sure to also regenerate the preprocessed source files (cd into `parser/preprocessed` and run the `genfiles.sh` shell script). Our build system uses flex and bison to make the lexer and parser when they are available, but it has the preprocessed files as a fallback if flex and/or bison aren't installed.